.travis.yml

```diff
@@ -12,3 +12,5 @@ before_install:
   - export CHROME_BIN=chromium-browser
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
I'm not sure we **need** to install yarn for Travis, I think they deliver it with [their linux images](https://travis-ci.org/github/zxing-js/library/jobs/694760818#L204).

Reply:
I thought so too, but actually build fails with a too old yarn: https://travis-ci.org/github/HaithemMosbahi/ngx-avatar/builds/694470294#L499

Reply:
That's odd, haha. I don't know how my build pipe works then, we can let it there for now anyways and improve if possible with time. Maybe Travis auto-includes yarn once it detects `yarn.lock`. Let's see what happens.